pcieutil/main.py

```diff
@@ -181,8 +181,10 @@ def pcie_check():
             log_warning("PCIe Device: " +  item["name"] + " Not Found")
             err+=1
     if err:
+        os.system('redis-cli -n 6 SET "PCIE_STATE|system" "0"')
         print "PCIe Device Checking All Test ----------->>> FAILED"
     else:
+        os.system('redis-cli -n 6 SET "PCIE_STATE|system" "1"')
```

Comment:
While I'm not sure if we should update the DB here, if we do decide to take this approach, calling `redis-cli` is not the proper method. You should import the swss-common library and use it to interact with the DB.